ovos_core/intent_services/__init__.py

```
@@ -408,26 +411,33 @@ def handle_utterance(self, message: Message):
+                        if match.skill_id and match.skill_id in sess.blacklisted_skills:
+                            LOG.debug(
+                                f"ignoring match, skill_id '{match.skill_id}' blacklisted by Session '{sess.session_id}'")
+                            continue
+                        if isinstance(match, IntentHandlerMatch) and match.match_type in sess.blacklisted_intents:
+                            LOG.debug(
+                                f"ignoring match, intent '{match.match_type}' blacklisted by Session '{sess.session_id}'")
+                            continue
+                        try:
+                            self._emit_match_message(match, message, intent_lang)
+                            break
+                        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)